ports/analog/boards/max32690evkit/board.c

```diff
@@ -0,0 +1,59 @@
+// This file is part of the CircuitPython project: https://circuitpython.org
+//
+// SPDX-FileCopyrightText: Copyright (c) 2024 Brandon Hurst, Analog Devices, Inc
+//
+// SPDX-License-Identifier: MIT
+
+#include "supervisor/board.h"
+#include "supervisor/port.h"
+#include "mpconfigboard.h"
+#include "max32_port.h"
+
+// Board-level setup for MAX32690
+mxc_gpio_regs_t *gpio_ports[NUM_GPIO_PORTS] =
+{MXC_GPIO0, MXC_GPIO1, MXC_GPIO2, MXC_GPIO3, MXC_GPIO4};
+
+// clang-format off
+const mxc_gpio_cfg_t pb_pin[] = {
+    { MXC_GPIO4, MXC_GPIO_PIN_0, MXC_GPIO_FUNC_IN, MXC_GPIO_PAD_NONE, MXC_GPIO_VSSEL_VDDIOH, MXC_GPIO_DRVSTR_0},
+};
+const int num_pbs = (sizeof(pb_pin) / sizeof(mxc_gpio_cfg_t));
+
+const mxc_gpio_cfg_t led_pin[] = {
+    { MXC_GPIO0, MXC_GPIO_PIN_14, MXC_GPIO_FUNC_OUT, MXC_GPIO_PAD_NONE, MXC_GPIO_VSSEL_VDDIO, MXC_GPIO_DRVSTR_0 },
+    { MXC_GPIO2, MXC_GPIO_PIN_12, MXC_GPIO_FUNC_OUT, MXC_GPIO_PAD_NONE, MXC_GPIO_VSSEL_VDDIO, MXC_GPIO_DRVSTR_0 },
+};
+const int num_leds = (sizeof(led_pin) / sizeof(mxc_gpio_cfg_t));
+// clang-format on
+
+// DEFAULT:  Using the weak-defined supervisor/shared/board.c functions
+
+/***** OPTIONAL BOARD-SPECIFIC FUNCTIONS from supervisor/board.h *****/
+// Returns true if the user initiates safe mode in a board specific way.
+// Also add BOARD_USER_SAFE_MODE in mpconfigboard.h to explain the board specific
+// way.
+// bool board_requests_safe_mode(void);
+
+volatile uint32_t system_ticks = 0;
+
+void SysTick_Handler(void) {
+    system_ticks++;
+}
+
+uint32_t board_millis(void) {
+    return system_ticks;
+}
```

Comment:
Why is this board specific?

Reply:
It seems unusual to have to provide `board_millis`. Unless I'm mistaken, this identifier is not otherwise used in CircuitPython. Rather, it seems it may be defined & used in tinyusb:
```c
// lib/tinyusb/hw/bsp/max32690/family.c
#if CFG_TUSB_OS == OPT_OS_NONE
volatile uint32_t system_ticks = 0;

void SysTick_Handler(void) {
  system_ticks++;
}

uint32_t board_millis(void) {
  return system_ticks; 
}
#endif
```
Perhaps there's a mistake with the `#defines` for tinyusb?

Reply:
Not sure where I grabbed this from, must've conflated the board requirements with tinyusb. Removed for both 'apard32690' and 'max32690evkit' board targets in the latest commit.